menderqa_pipeline.go

```diff
@@ -522,10 +415,10 @@ func getBuildParameters(
 			Value: &readHead,
 		})
 
-	return getClientBuildParameters(buildParameters, build)
+	return _getClientBuildParameters(buildParameters, build)
```

Comment:
:sob: ,
This is pretty ugly. Maybe rename to `getClientAcceptanceTestsParameters` or similar. Once we get the rid of Mender 3.7 we could clean this further and squash the two functions.